src/Chan.cpp

```diff
@@ -262,6 +262,11 @@ void CChan::InheritAutoClearChanBuffer(bool b) {
 	}
 }
 
+void CChan::ResetAutoClearChanBuffer() {
+	SetAutoClearChanBuffer(m_pNetwork->GetUser()->AutoClearChanBuffer());
```

Comment:
is a null check necessary? is it possible to have a floating CChan instance that is not associated to a network? the constructor already assumes that a valid network is passed